central/metadata/service/service_impl.go

```diff
@@ -35,8 +36,13 @@ var (
 			"/v1.MetadataService/GetDatabaseBackupStatus",
 			"/v1.MetadataService/GetCentralCapabilities",
 		},
-		allow.Anonymous(): {
+		// When this endpoint was public, Sensor has been relying on it to check
+		// Central's availability. While Sensor might not do so today, we need
+		// to ensure backward compatibility with older Sensors.
+		or.SensorOrAuthorizer(user.With()): {
```

Comment:
super nit: it does look a bit confusing in terms of syntax (with the addition of the comment on `With`), might be worthwhile to create a handy wrapper that only does the identity check called `user.Authenticated()`?

Reply:
Yeah, I agree, however, it seems to be a common pattern in our codebase: grepping for `or.SensorOrAuthorizer(` yields 8 results. Introducing `user.Authenticated()` might be a tricky abstraction: will it include sensor, scanner, collector, admission controller as well? Are they "users"? Do we even want to allow all ACS components to be treated the same?

I thought about using `Or()` directly but we don't seem to have precedence for such cases (with scanner or sensor) in the codebase. 

What I can offer is renaming `SensorOrAuthorizer()` to `SensorOr()`:
```
		or.SensorOr(user.With()): {
```

Would it improve readability from your perspective?

Reply:
Yeah It would improve readability a bit, I think what I was looking for was `user.AuthenticatedUsersOnly()` similar to the `SensorOnly()` or `CollectorOnly()` ones. 

Supposedly the `user.XX` ones only assume non-service identities, right? Technically its possible, but I don't think any of the service identities have roles and permissions associated with them, right? (hence also the need for the construct of `ScannerOr` or `SensorOr`).

Either way this is optional for this PR, so I'm fine without bigger refactorings and be consistent with the naming of `ScannerOr`, hence naming `SensorOrAuthorizer` to `SensorOr`.